pkg/apiserver/api_installer.go

```diff
@@ -148,6 +171,19 @@ func (a *APIInstaller) registerResourceHandlers(path string, storage RESTStorage
 
 		itemPath := path + "/{name}"
 		nameParams := append(params, nameParam)
+		namespaceFn = func(req *restful.Request) (namespace string, err error) {
+			return
```

Comment:
Right. It's difficult to tell at first glance that most (or all?) of the naked returns are inside tiny functions here.